static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -57,30 +76,54 @@ function SentryFunctionDetails(props: Props) {
 
   const handleSubmitSuccess = data => {
     addSuccessMessage(t('Sentry Function successfully saved.', data.name));
+    const baseUrl = `/settings/${orgId}/developer-settings/sentry-functions/`;
+    // TODO: should figure out where to redirect this
+    const url = `${baseUrl}${data.slug}/`;
+    if (sentryFunction) {
+      addSuccessMessage(t('%s successfully saved.', data.name));
+    } else {
+      addSuccessMessage(t('%s successfully created.', data.name));
+    }
+    browserHistory.push(url);
   };
 
   function handleEditorChange(value, _event) {
     form.current.setValue('code', value);
   }
 
-  const defaultCode = `exports.yourFunction = (req, res) => {
-    let message = req.query.message || req.body.message || 'Hello World!';
-    console.log('Query: ' + req.query);
-    console.log('Body: ' + req.body);
-    res.status(200).send(message);
-};`;
+  async function handleDelete() {
+    try {
+      await api.requestPromise(endpoint, {
+        method: 'DELETE',
+      });
+      addSuccessMessage(t('Sentry Function successfully deleted.'));
+      // TODO: Not sure where to redirect to, so just redirect to the unbuilt Sentry Functions page
+      browserHistory.push(`/settings/${orgId}/developer-settings/sentry-functions/`);
+    } catch (err) {
+      addErrorMessage(t(err.responseJSON.detail));
```

Comment:
@vpandiarajan20 just to be safe, we should do something like `err?.responseJSON?.detail || 'Unkown'`. Also, we can't translate dynamic strings.